client/src/pages/users/UserRow.js

```diff
@@ -0,0 +1,16 @@
+import { Td, Tr } from "@chakra-ui/react";
+import React from "react";
+
+const UserRow = ({ user }) => {
+	return (
+		<Tr>
+			<Td>{user.name}</Td>
+			<Td>{user.email}</Td>
+			<Td>PLACEHOLDER</Td>
```

Comment:
This is something that you are going to implement later?

Reply:
Yes, I had to do a bit of research into auth0 because they don't provide a way to pull roles/permissions in the same request as users.

Unfortunately there's no way to do it, and I implemented a way to pull roles of each user separately (11 requests if we have 10 users :( )

The performance is really slow, so I'm not sure if this is the right way - we will definitely need server side pagination if we are going through with this